test/parallel/test-stream2-readable-bytesread.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+require('../common');
+var assert = require('assert');
+
+var Readable = require('stream').Readable;
+var Duplex = require('stream').Duplex;
+var Transform = require('stream').Transform;
+
+(function() {
+  var readable = new Readable({
+    read: function(n) {
+      var i = this._index++;
+      if (i > this._max)
+        this.push(null);
+      else {
+        this.push(new Buffer('a'));
```

Comment:
Ok, I could probably live with it. However I don't really enjoy using
capital letter function names, as they are usually used only for naming
classes.

On Thursday, December 31, 2015, Trevor Norris <notifications@github.com>
wrote:

> In test/parallel/test-stream2-readable-bytesread.js
> <https://github.com/nodejs/node/pull/4372#discussion_r48672201>:
>
> > +
> > +require('../common');
> > +var assert = require('assert');
> > +
> > +var Readable = require('stream').Readable;
> > +var Duplex = require('stream').Duplex;
> > +var Transform = require('stream').Transform;
> > +
> > +(function() {
> > +  var readable = new Readable({
> > +    read: function(n) {
> > +      var i = this._index++;
> > +      if (i > this._max)
> > +        this.push(null);
> > +      else {
> > +        this.push(new Buffer('a'));
>
> @indutny <https://github.com/indutny> I'm not sure what's constraining
> about it, and using new creates an unnecessary allocation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/4372/files#r48672201>.
>